msg/Spatz/SystemParams.msg

```diff
@@ -0,0 +1,16 @@
+float64 width
+float64 length
+float64 origin_x
+float64 axes_distance
+float64 mass
+
+float64 inertia
+float64 max_steering_angle
+float64 axes_moment_ratio
+
+
+float64 dist_cog_to_front_axle
+float64 dist_cog_to_rear_axle
+float64 dist_cam_origin_x
```

Comment:
cam=camera? muss das hier drin stehen? die genaue kamera-position wird eigentlich erst durch die extrinsische calib festgestellt, und dann in kamera-params bzw über tf published